### ISSUE TITLE
The graphqls extension is not accepted as an introspection output

### ISSUE THREAD

**Issue by steebchen** [kamilkisiela/graphql-inspector#268]
While I can use the .graphqls extension for documents, I can't use it for the schema file in the validate command:

```
$ graphql-inspector validate "src/**/*.graphqls" schema.graphql
success All documents are valid
```

```
$ graphql-inspector validate "src/**/*.graphql" schema.graphqls
error Unable to find matching files for glob: schema.graphqls in directory: .
```

**Comment by kamilkisiela** [comment]
Do you have any public repo where I could check it?

I tried in `/example` and worked:
<img width="1007" alt="Zrzut ekranu 2019-05-15 o 15 56 26" src="https://user-images.githubusercontent.com/8167190/57781201-03f17b80-772a-11e9-8635-0f4d1c52c51f.png">

**Comment by steebchen** [comment]
Weird, I can't reproduce it either right now. I recently re-installed the cli so maybe that was the issue or something was wrong on my end. Thanks a lot for checking.

**Comment by steebchen** [comment]
The `introspect` command does not accept .graphqls files, though. Maybe I confused the command while creating the issue.

```
$ graphql-inspector introspect http://localhost:4000/graphql --write schema.graphqls
error Only .graphql, .gql and .json files are supported
```

**Comment by kamilkisiela** [comment]
Released as `v1.19.0`

**Comment by steebchen** [comment]
Thanks so much for fixing!
